src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlCommandTest.cs

```diff
@@ -136,6 +136,19 @@ public void Constructor3()
             Assert.Equal(UpdateRowSource.Both, cmd.UpdatedRowSource);
         }
 
+        [Theory]
+        [InlineData(0)]
+        [InlineData(30)]
+        [InlineData(15)]
+        public void Constructor3_CommandTimeout(int timeout)
+        {
+            SqlConnection conn = new SqlConnection($"Command Timeout = {timeout}");
+            SqlCommand cmd;
+
+            cmd = new SqlCommand(COMMAND_TEXT, conn);
+            Assert.Equal(timeout, cmd.CommandTimeout);
```

Comment:
Can you extend this test to check if setting a new value for the `CommandTimeout `property of the `SqlCommand `object will successfully override the value set in the connection string?